### ISSUE TITLE
Log in on web ui and scan is performed on cliet side.

### ISSUE THREAD

**Issue by simoher** [AHAAAAAAA/PokemonGo-Map#2960]
## Expected Behavior

Usee can log in to own acc and contribrute to the scanning af new areas, without adding load to the server. This could be a move forward for a db where users are contribruteing, instead of running thir own scanning server. How much would it requier to just use the same db like that?
I think the node.js version is implemeting this. 

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Your Environment

* Version used:
* Environment name and version (e.g. Python 2.7):
* Operating System and version (desktop or mobile):

**Comment by prusswan** [comment]
There might be trust issues, so I will assume that the server only receives and collates the search results from the users but not their credentials.

**Comment by simoher** [comment]
Yes. That were allso one of the points of lettimg users scannimg thor own locations. 
I also seep the location as an trust issue, but i think it's obvius when you press yes to share your location. 
Maybe the scanner should start within some random distamce  to the users exat location, to blur theier location.
